packages/application-generic/src/usecases/map-trigger-recipients/map-trigger-recipients.use-case.ts

```diff
@@ -115,42 +113,29 @@ export class MapTriggerRecipients {
     userId: UserId,
     recipients: TriggerRecipients
   ): Promise<ISubscribersSource[]> {
-    const featureFlagCommand = FeatureFlagCommand.create({
-      environmentId,
-      organizationId,
-      userId,
-    });
-    const isEnabled = await this.getIsTopicNotificationEnabled.execute(
-      featureFlagCommand
-    );
+    const topics = this.findTopics(recipients);
 
-    if (isEnabled) {
-      const topics = this.findTopics(recipients);
-
-      const subscribers: ISubscribersSource[] = [];
-
-      for (const topic of topics) {
-        const getTopicSubscribersCommand = GetTopicSubscribersCommand.create({
-          environmentId,
-          topicKey: topic.topicKey,
-          organizationId,
-        });
-        const topicSubscribers = await this.getTopicSubscribers.execute(
-          getTopicSubscribersCommand
-        );
-
-        topicSubscribers.forEach((subscriber: TopicSubscribersDto) =>
-          subscribers.push({
-            subscriberId: subscriber.externalSubscriberId,
-            _subscriberSource: SubscriberSourceEnum.TOPIC,
-          })
-        );
-      }
-
-      return subscribers;
+    const subscribers: ISubscribersSource[] = [];
+
+    for (const topic of topics) {
+      const getTopicSubscribersCommand = GetTopicSubscribersCommand.create({
+        environmentId,
+        topicKey: topic.topicKey,
+        organizationId,
+      });
+      const topicSubscribers = await this.getTopicSubscribers.execute(
+        getTopicSubscribersCommand
+      );
+
+      topicSubscribers.forEach((subscriber: TopicSubscribersDto) =>
+        subscribers.push({
+          subscriberId: subscriber.externalSubscriberId,
+          _subscriberSource: SubscriberSourceEnum.TOPIC,
+        })
+      );
     }
 
-    return [];
+    return subscribers;
```

Comment:
This is the small change required to remove the `IS_TOPIC_NOTIFICATION_ENABLED` FF disabled path.

Reply:
I think we can this conversation here https://github.com/novuhq/novu/pull/5100/files#r1459396557